web/js/diceware/random-integer.js

```diff
@@ -0,0 +1,36 @@
+export { getLocalIntegers };
+
+async function getLocalIntegers(count,min,max) {
```

Comment:
Perhaps these defaults are better applied in the function header, like `count = 2`, etc.